packages/@aws-cdk/cdk/lib/cloudformation/stack.ts

```diff
@@ -125,6 +130,54 @@ export class Stack extends Construct {
     return r as Resource;
   }
 
+  public synthesize(session: ISynthesisSession): void {
```

Comment:
Do we want this to be public, re: autocomplete?

As in, if I type `myStack.`, do I want framework methods to pop up in my autcomplete?

Reply:
Not really. It was supposed to be protected. Let me check.

Reply:
Fixed